docs/src/main/asciidoc/transaction.adoc

```diff
@@ -3,17 +3,21 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
-= Using Transactions in Quarkus
+= Using transactions in Quarkus
 include::_attributes.adoc[]
-:categories: data
-:summary: Quarkus comes with a Transaction Manager and uses it to coordinate and expose transactions to your applications. Each extension dealing with persistence will integrate with it for you, and you will explicitly interact with transactions via CDI. This guide will walk you through all that.
+:diataxis-type: reference
+:categories: data,getting-started
 :topics: data,jpa,jta,transactions,narayana
 :extensions: io.quarkus:quarkus-narayana-jta
 
-Quarkus comes with a Transaction Manager and uses it to coordinate and expose transactions to your applications.
-Each extension dealing with persistence will integrate with it for you.
-And you will explicitly interact with transactions via CDI.
-This guide will walk you through all that.
+Quarkus includes a Transaction Manager extension used to coordinate and expose transactions to your applications as described in the link: https://jakarta.ee/specifications/transactions/[Jakarta Transactions] specification, formerly known as Java Transaction API (JTA).
```

Comment:
May I ask why you added "extension" to "Transaction Manager"? Could you reference it?

Reply:
I tend to agree with @jmfinelli this is not the best wording, since there is no Quarkus extension named even remotely like that (the extension is called `quarkus-narayana-jta`, for better or worse). Plus "Transaction Manager extension" can be interpreted as "something that extends a transaction manager", which would be incorrect.

If you want to mention that extensions are involved, I guess this could work:

```suggestion
The `quarkus-narayana-jta` extension provides a Transaction Manager that coordinates and expose transactions to your applications as described in the link: https://jakarta.ee/specifications/transactions/[Jakarta Transactions] specification, formerly known as Java Transaction API (JTA).
```